test/applications_tests.coffee

```diff
@@ -198,36 +259,36 @@ describe "Applications management", ->
                 request.method.should.equal "POST"
 
             it "And the proxy have been requested to update its routes", ->
-                @proxy.lastCall().request.url.should.equal "/routes/reset"    
-
-
-    ###describe "Application auto stop", ->
-
-        before resetTestServers
-
-        describe "POST /api/applications/:slug/start Start an app", ->
-
-            it "When I send a request to stop an application", (done) ->
-                @client.post "api/applications/my-app/start", {}, done
-
-            it "Then it sends me a success response with the updated app", ->
-                @response.statusCode.should.equal 200
-                expect(@body.success).to.be.ok
-                expect(@body.app.state).to.equal 'installed'
+                @proxy.lastCall().request.url.should.equal "/routes/reset"
 
-            it "And I wait 3 minutes", (done) ->
-                @timeout(3.2 * 60 * 1000)
-                setTimeout () ->
+            it 'Then application has an access', (done) ->
+                Application.all key:"my-app", (err, apps) =>
+                    ds = new Client "http://localhost:9101/"
+                    ds.setBasicAuth 'home', 'token'
+                    ds.post 'request/access/byApp/', key: apps[0].id, (err, res, body) =>
+                        @password = body[0].value.token
+                        done()
+
+            it 'Then application can access to contact', (done) ->
+                ds = new Client "http://localhost:9101/"
+                ds.setBasicAuth 'my-app', @password
+                data =
+                    docType: 'contact'
+                    slug: "blabla"
+                ds.post 'data/', data, (err, res, body) ->
+                    should.not.exist body.error
                     done()
-                , 3.1 * 60 * 1000
-
-            it "And controller have been requested to stop this app", ->
-                request = @controller.lastCall().request
-                request.url.should.equal "/drones/my-app/stop"
-                request.method.should.equal "POST"
 
-            it "And the proxy have been requested to update its routes", ->
-                @proxy.lastCall().request.url.should.equal "/routes/reset"  ###
+            it 'Then application can access to contact', (done) ->
```

Comment:
We test application permissions after : 
* installation
* update
* start/stop
Application document or access document can be changed after an update or restart (as an example : password are changed after a restart).